src/features/import/utils/prepareImportOperations.ts

```diff
@@ -109,6 +109,17 @@ export default function prepareImportOperations(
           });
         }
 
+        if (column.kind === ColumnKind.ENUM) {
+          column.mapping.forEach((mappedColumn) => {
+            if (mappedColumn.value === row.data[colIdx] && mappedColumn.key) {
+              personImportOps[rowIndex].data = {
+                ...personImportOps[rowIndex].data,
+                [`${column.field}`]: mappedColumn.key,
```

Comment:
This should be translated to `null` if `""` (could also be solved elsewhere, e.g. closer to the UI, but doing it here should make it testable).

Reply:
How are you thinking about this @niklasva82? Is this case not relevant, i.e. can `mappedColumn.key` never be "" (because that's not how blanks are handled)?

Reply:
Keys will never be an empty string because the API doesn't allow such keys to be configured, and  the value is always null on empty values because `selectOption` is called with `(key, null)` in `EnumConfig.tsx:82`

You're commenting old code here though?

Reply:
Ok, cool! I was just asking because you hadn't adressed this comment. Now I know, and I agree we can resolve this. 💯